### PR TITLE
Reneiski_Homework2

### PR DESCRIPTION
**Pull request by agrren**
Here is my 2nd homework!
Could you be so kind and review it!
Thank you in advance!

### CODE REVIEW

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 1]
```diff
@@ -1,5 +1,205 @@
 ﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hi! What's your name?");
-string name = Console.ReadLine();
-Console.WriteLine("Nice to meet you, " + name + "!");
-Console.ReadLine();
+//Console.WriteLine("Hi! What's your name?");
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: You may remove this code

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 2]
```diff
@@ -1,5 +1,205 @@
...
+//string name = Console.ReadLine();
+//Console.WriteLine("Nice to meet you, " + name + "!");
+//Console.ReadLine();
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Task1();
+    }
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: add empty line after method here and for all cases below

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 3]
```diff
@@ -1,5 +1,205 @@
...
+            var numberOfYears = number / 365;
+            var numberOfMonths = (number - numberOfYears * 365) / 30;
+            var numberOfDays = number - numberOfMonths * 30 - numberOfYears * 365;
+            Console.WriteLine($"Number of years: {numberOfYears}");
+            Console.WriteLine($"Number of month: {numberOfMonths}");
+            Console.WriteLine($"Number of days: {numberOfDays}");
+            Console.ReadLine();
+        }
+    }
+
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: leave only one empty line after method

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 4]
```diff
@@ -1,5 +1,205 @@
...
+                case ('-'):
+                    Console.WriteLine($"The diff is: {A - B}");
+                    break;
+                case ('*'):
+                    Console.WriteLine($"The prod is: {A * B}");
+                    break;
+                case ('/'):
+                    Console.WriteLine($"The div is: {A / B}");
+                    break;
+
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: remove empty line

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 5]
```diff
@@ -1,5 +1,205 @@
...
+                    break;
+                case ('/'):
+                    Console.WriteLine($"The div is: {A / B}");
+                    break;
+                
+            }
+            Console.ReadLine();
+        }
+    }
+
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: remove 2 empty lines

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 6]
```diff
@@ -1,5 +1,205 @@
...
+                Console.WriteLine("The number is odd");
+            Console.ReadLine();
+        }
+    }
+    private static void Task8()
+    {
+        while (true)
+        {
+            Console.Clear();
+            double A;
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: please use camelCase for local variable (should be double a). Please correct here and below for other variables

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 7]
```diff
@@ -1,5 +1,205 @@
...
+        Console.WriteLine($"Variable: {num += 5}");
+        Console.ReadLine();
+    }
+    private static void Task2()
+    {
+        while (true)
+        {
+            Console.Clear();
+            ushort number;
+            try
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: It's better to have empty line before try and after catch\finally. Please correct here and across the file

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 8]
```diff
@@ -1,5 +1,205 @@
...
+
+        Console.WriteLine(minus34);
+        Console.WriteLine(prosto4);
+        Console.WriteLine(hello);
+        Console.WriteLine(letterR);
+        Console.WriteLine(numberStochkoi);
+        Console.WriteLine(sorokTysach);
+        Console.WriteLine(istynno);
+        Console.WriteLine(zero);
+
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: please remove empty line

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 9]
```diff
@@ -1,5 +1,205 @@
...
+    }
+    private static void Task5()
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: In this task you assign variabled by yourself, but it's needed to get them from the user

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 10]
```diff
@@ -1,5 +1,205 @@
...
+    }
+    private static void Task1()
+    {
+        var num = 20;
+        Console.WriteLine($"Variable: {num += 5}");
+        Console.ReadLine();
+    }
+    private static void Task2()
+    {
+        while (true)
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Do you really need while(true) here? If so, could you please explain why?

File: Reneiski_Homework1/Reneiski_Homework1/Properties/launchSettings.json [hunk 11]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Properties/launchSettings.json]: I don't think you need this file and folder 'Properties' in general...

File: Reneiski_Homework1/Reneiski_Homework1/Reneiski_Homework1.csproj.user [hunk 12]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Reneiski_Homework1.csproj.user]: I don't think you need to commit and merge this file

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 1 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 2 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 10 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 7 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 3 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 8 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 9 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 6 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 4 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs (hunk 5 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Program.cs]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Properties/launchSettings.json (hunk 11 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Properties/launchSettings.json]: Fixed

Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Properties/launchSettings.json]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Reneiski_Homework1.csproj.user (hunk 12 above)
Comment (agrren) [reply Reneiski_Homework1/Reneiski_Homework1/Reneiski_Homework1.csproj.user]: Fixed

File: Reneiski_Homework1/Reneiski_Homework1/Program.cs [hunk 13]
```diff
@@ -1,5 +1,9 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hi! What's your name?");
-string name = Console.ReadLine();
-Console.WriteLine("Nice to meet you, " + name + "!");
-Console.ReadLine();
+﻿using Reneiski_Homework1.Lecture3;
```
Comment (ashurakova) [inline Reneiski_Homework1/Reneiski_Homework1/Program.cs]: you may remove this namespace because it's not used

**Pull request merged**